m4_sessions/1803-inning/group_4.md

```diff
@@ -1,13 +1,46 @@
-Group 4
+## Group 4 Finding Public Data and Using it to Change the World
 
-Group Members:
+### Group Members:
+- Nora Gully
+- Kelley Jenkins
+- Casey Dallavalle
+- Josi McClellan
 
-Name of Session:
+### Name of Session: 
+- Finding Public Data and Using it to Change the World
 
-Session Description (Why should students come to this session):
+### Session Description (Why should students come to this session):
+- To learn about publicly available APIs that they can use with on their personal projects.
 
-Session Outcomes (What will students learn/leave with?):
+### Session Outcomes (What will students learn/leave with?):
```

Comment:
This is AWESOME!  I love that this is focused on the mission and provides tangible technical ways to implement it!